integration/test_flow_integration.py

```diff
@@ -144,10 +144,10 @@ def _get_sql_by_key_all_attrs(
             if i != 0:
                 where_statement += " and "
             if sql_table.columns[key_name[i]].type.python_type == str:
-                where_statement += f'{key_name[i]}="{key[i]}"'
+                where_statement += f'test_table.[{key_name[i]}]="{key[i]}"'
             else:
-                where_statement += f"{key_name[i]}={key[i]}"
-    query = f"SELECT * FROM {sql_table} where ({where_statement})"
+                where_statement += f"test_table.[{key_name[i]}]={key[i]}"
```

Comment:
Which sql db are you using?
Not all of them support '[]' to describe column name

Reply:
you need to change here to by the relevant db- [] for mysql and "" for non-mysql.